README.md

```diff
@@ -18,11 +18,16 @@ The master branch is in version 1.5.0-SNAPSHOT:
 
 ### Dependency Information
 
+Alluxio provides several different client interfaces. The Alluxio file system interface provided by
+`alluxio-core-client-fs` is suggested for performance and flexibility. If you want to use other
+interfaces, include the appropriate client artifact. For example, `alluxio-core-client-hdfs` provides
+a client implementing HDFS's file system API.
```

Comment:
The uber jar is a runtime dependency not compile time so we omit it here (same reason why we did not talk about `core-client-with-depenedencies` before.